content/en/blog/_posts/2024-08-20-websocket-transition.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: blog
+title: 'Streaming Transitions from SPDY to WebSockets'
```

Comment:
Should the title mention Kubernetes 1.31? A typical post-release article does.

/lgtm cancel

Reply:
It should be: 
`title: "Kubernetes 1.31: Streaming Transitions from SPDY to WebSockets"`

Reply:
Fixed.